common/src/main/java/org/apache/rocketmq/common/protocol/RequestCode.java

```diff
@@ -17,7 +17,9 @@
 
 package org.apache.rocketmq.common.protocol;
 
-public class RequestCode {
+import org.apache.rocketmq.remoting.protocol.RemotingSysRequestCode;
+
```

Comment:
In fact, we should remove RemotingSysRequestCode which is part of earlier StartTLS implementation and is no longer in use.